Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance_README.md

```diff
@@ -10,15 +10,16 @@ This playbook does not use any sub-playbooks.
 
 ### Integrations
 
-This playbook does not use any integrations.
+* Cortex Attack Surface Management
 
 ### Scripts
 
-Set
+* Set
```

Comment:
```suggestion
Set
```
When there is only one item, we do not bullet it.

Reply:
Addressed in a1e7cfb